other/upgrade_2-1_mysql.sql

```diff
@@ -1046,7 +1046,7 @@ if (!empty($select_columns))
 				'{db_prefix}themes',
 				array('id_member' => 'int', 'id_theme' => 'int', 'variable' => 'string', 'value' => 'string'),
 				$inserts,
-				array('id_theme', 'id_member', 'variable')
+				array('id_member', 'id_theme', 'variable', 'value')
```

Comment:
value field can't be a pk field.
and the pk is defined as 'id_theme', 'id_member', 'variable' -> i would keep the old arrengment